packages/xrpl/test/models/batch.test.ts

```diff
@@ -90,7 +90,7 @@ describe('Batch', function () {
             Fee: '0',
             Flags: 0x40000000,
             NetworkID: 21336,
-            Sequence: 0,
+            Sequence: 1,
```

Comment:
There is a similar error on line-53. Two inner batch transactions have the identical sequence number.

Reply:
I was of the understanding that rippled fills a "phantom" sequence number for the inner batch-transactions, is that not the case? Do users need to fill up the seq numbers for all the inner batch transactions?

Reply:
Correct

Reply:
can you also rectify this error within this PR? its a small change
> There is a similar error on line-53. Two inner batch transactions have the identical sequence number.